simmechanics/data/icub2_5/conf/gazebo_icub_ft_imu_3B14.ini

```diff
@@ -0,0 +1,7 @@
+[include "gazebo_icub_robotname.ini"]
+
+name /${gazeboYarpPluginsRobotName}/r_foot_ft_imu
+period 0.01
+
+device inertial
+subdevice gazebo_imu
```

Comment:
Related gazebo-yarp-plugins issue: https://github.com/robotology/gazebo-yarp-plugins/issues/383 .

Reply:
At the moment, since the MAS plugin in Gazebo is still under development, many of us are using the workaround of using the analog sensor interface. 

But then since, it's user-specific, we could remove these files until the MAS plugin is integrated.